tests/unit/Alerts.spec.ts

```diff
@@ -0,0 +1,137 @@
+// Libraries
+import Vue from 'vue'
+import Vuetify from 'vuetify'
+import { shallowMount, Wrapper } from '@vue/test-utils'
+import { createPinia, setActivePinia } from 'pinia'
+import Alerts from '@/components/Dashboard/Alerts.vue'
+
+// Alerts
+import Amalgamation from '@/components/Dashboard/Alerts/Amalgamation.vue'
+import FrozenInformation from '@/components/Dashboard/Alerts/FrozenInformation.vue'
+import MissingInformation from '@/components/Dashboard/Alerts/MissingInformation.vue'
+import NotInCompliance from '@/components/Dashboard/Alerts/NotInCompliance.vue'
+import NotInGoodStanding from '@/components/Dashboard/Alerts/NotInGoodStanding.vue'
+
+Vue.use(Vuetify)
+
+const vuetify = new Vuetify({})
+setActivePinia(createPinia())
+
+describe('Dashboard - UI', () => {
+  let wrapper: Wrapper<Vue>
+
+  beforeEach(() => {
+    // create wrapper for Dashboard
+    // this stubs out the 5 sub-components
+    wrapper = shallowMount(Alerts, {
```

Comment:
This test suite verifies the Alert component only. There are other test suites that verify each alert.